app/styles/app/modules/buttons.scss

```
@@ -257,6 +258,10 @@
   }
 }
 
+.action-button--priortize{
```

There's a typo in the class name:
```suggestion
.action-button--prioritize {
```

---

Done.